### ISSUE TITLE
How to ignore constructor parameters?

### ISSUE THREAD

**Issue by ziofat** [edvardchen/eslint-plugin-i18next#53]
Like this case:

```
throw new Error('some error message');
```

We do not throw error to users so this text should be ignored. But ignoreCallee seems not working with constructor.

**Comment by carolinamartes** [comment]
I also have the same issue.

**Comment by SkariDamm** [comment]
Same here, love this plugin though 😄
